lib/internal/streams/from.js

```diff
@@ -107,6 +107,10 @@ function from(Readable, iterable, opts) {
       break;
     }
   }
+
+  readable.on('pipe', readable.resume.bind(readable));
+  readable.on('unpipe', readable.pause.bind(readable));
```

Comment:
Spent over 5 hours just for these 2 lines 😬

Reply:
I don't think this is the correct fix for this issue. `resume` and `pause` should be automatically handled my the machinery during pipe and unpipe.

Why do you think so?

@ronag wdyt?

Reply:
Yea, this is wrong.